homeassistant/components/august/lock.py

```diff
@@ -15,6 +15,8 @@
 
 SCAN_INTERVAL = timedelta(seconds=10)
 
+PARALLEL_UPDATES = 0
```

Comment:
Setting parallel updates to 0 means that all requests for all entities are happening at the same time. State updates but also requests to lock/unlock locks. Can the API handle that ?

Reply:
Just found _get_parallel_updates_semaphore.  I think it should actually be PARALLEL_UPDATES = 1 so only one entity can update at a time

Reply:
1 is the default.